jwst/cube_build/ifu_cube.py

```diff
@@ -1324,13 +1325,15 @@ def setup_ifucube_wcs(self):
                 lambda_max.append(lmax)
     # ________________________________________________________________________________
     # done looping over files determine final size of cube
+        corner_a = np.array(corner_a)
+        corner_a = wrap_ra(corner_a)
 
         final_a_min = min(corner_a)
         final_a_max = max(corner_a)
         final_b_min = min(corner_b)
         final_b_max = max(corner_b)
 
-        log.debug(f'final a and b:{final_a_min, final_b_min, final_a_max, final_b_max}')
+        log.info(f'final a and b:{final_a_min, final_b_min, final_a_max, final_b_max}')
         log.debug(f'final wave:   {min(lambda_min), max(lambda_max)}')
```

Comment:
If a and b values are important enough to warrant always logging in an info level message, would the same apply to final wave (i.e. should that also be promoted from debug to info level)?